internal/config/production.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"github.com/layer5io/meshery-adapter-library/common"
+	configprovider "github.com/layer5io/meshery-adapter-library/config/provider"
+)
+
+var (
+	ProductionConfig = configprovider.Options{
+		ServerConfig:   productionServerConfig,
+		MeshSpec:       productionMeshSpec,
+		ProviderConfig: productionProviderConfig,
+		Operations:     productionOperations,
+	}
+
+	productionServerConfig = map[string]string{
+		"name":    "linkerd-adapter",
+		"port":    "10001",
+		"version": "v1.0.0",
+	}
+
+	productionMeshSpec = map[string]string{
+		"name":     "linkerd",
+		"status":   "none",
+		"traceurl": "none",
+		"version":  "none",
```

Comment:
I still don't think "none" is a good choice. An empty string also means "none". Code should be able to handle an empty string. In order to distinguish between "none" and an url, one has to test for the special string "none", then someone might decide that empty is great too, then one has to test for 2 conditions....

what about: 
status: undefined (a constant, like the other status constants).
version: default (possibly a constant).
url: empty

Reply:
I agree with the above ones yes. Do you think its good to add it on the `status` package?